subschemas/substrate-chain/src/typeDefs/democracy.ts

```diff
@@ -45,7 +52,7 @@ export default /* GraphQL */ `
 
   type DemocracyReferendum {
     index: String!
-    meta: String!
+    meta: String
```

Comment:
Small thing: I think here you can return empty string - but it's always a string. Why not keep the `required`?

Reply:
Hey @jdomingos 
Since I refactored `formatCallMeta(imageProposal?.registry.findMetaCall(imageProposal.callIndex).meta)` into the `getCallParams` function and call to that function depends on having a proposal image the returned object from `getCallParams` can be undefined and thus `meta`. Otherwise we have to extract the `meta` in every resolver.